src/ftva_etl/metadata/marc.py

```diff
@@ -211,21 +213,29 @@ def _get_first_stripped(subfields: list[str]) -> str:
     name_of_part = _get_first_stripped(title_statement.get_subfields("p"))
     number_of_part = _get_first_stripped(title_statement.get_subfields("n"))
 
+    # TODO: Add logic to get `production_type` information from Filemaker record.
```

Comment:
I don't think this TODO comment is relevant any more, since `is_series` is passed into this function.